src/Handlers/RedisHandler.php

```diff
@@ -0,0 +1,177 @@
+<?php
+
+namespace CodeIgniter\Settings\Handlers;
+
+use CodeIgniter\I18n\Time;
+use Config\Cache;
+use RuntimeException;
+
+/**
+ * Provides cache persistence for Settings.
+ * Uses ArrayHandler for storage to minimize cache calls.
+ */
+class RedisHandler extends ArrayHandler
+{
+    /**
+     * The Query Builder for the Settings table.
+     */
+    private BaseRedisHandler $handler;
+
+    /**
+     * Array of contexts that have been stored.
+     *
+     * @var ?string[]
```

Comment:
This type is unclear to me. Does this mean `string[]|null` or `array<string|null>`?

Reply:
This should read as `string[]|null`. `array<string|null>` would be `(string|null)[]`

Reply:
I copied from DatabaseHandler.php

Reply:
🤦‍♂️